### DISCUSSION TITLE
Mod Organizer 2 Support

### DISCUSSION THREAD

**Discussion by BeranabusBarnes** [blu3mania/npp-papyrus#61] [Q&A]
Is additional Mod Organizer 2 support under consideration? You can run Notepad++ from inside MO2 right now, and it works fine, but being able to compile without doing so would be great. It's not a major deal though.

**Comment by blu3mania** [comment]
Sorry, I misread your comment. AFAIK, MO2 uses VFS and there is no way to bypass it, so you will have to use MO2 to launch Notepad++. A major reason why I don't use it.
